src/config/dynamic-theme-fixes.config

```diff
@@ -14870,6 +14870,8 @@ meteo.imgw.pl
 
 INVERT
 .meteo-chart > div > canvas
+.ft-copyright div div img
+#map-container span
```

Comment:
Again I will complain... You inverted everything on the map - even nice icons are inverted.
Invert breaks Szczecin as it is now not visible because dark... I propose to invert map:
#map-container svg 😉

Reply:
![obraz](https://github.com/darkreader/darkreader/assets/56877029/f880afe2-22c2-4f7a-9f73-3f4de3851c12)
doesn't look it better? :)